packages/rs-sdk/src/platform/transition/builder.rs

```diff
@@ -9,6 +9,7 @@ use dpp::state_transition::StateTransition;
 use dpp::version::PlatformVersion;
 
 /// Trait for building state transitions
+#[allow(async_fn_in_trait)]
```

Comment:
#[async_trait]

Reply:
Changed; however, I'm not sure if this solves the issue or just hides it.